### PR TITLE
Represent more information

### PR DESCRIPTION
**Pull request by dekespo**
I think we should make HTML interactive with user with JavaScript (showing plots for different languages by selecting them, showing/hiding some information). What do you think?

### CODE REVIEW

File: benchmarker.py [hunk 1]
```diff
@@ -1,5 +1,4 @@
 import time
-import subprocess
```
Comment (naslundx) [inline benchmarker.py:2]: Ah, nice.

File: benchmarker.py [hunk 2]
```diff
@@ -184,17 +185,21 @@ def save_to_file(languages, items, results):
...
-	html = '<html><body><table style="text-align:right">'
-	heading = "<th></th>" + "".join([ '<th style="font-style:bold;text-align:center;padding:5px 20px;">' + next + "</th>" for next in items])
-	html += "<tr>" + heading + "</tr>"
-	for i in range(len(languages)):
-		row = "<td>" + languages[i] +  "</td>" + "".join(['<td style="padding:5px 20px;">' + "{:.3f}".format(next) + " s" + "</td>" for next in average_results[i]])
-		html += "<tr>" + row + "</tr>"
-	html += "</table></body></html>"
+	html = '<html><body>'
+	for measure in sorted(results[0][0].keys()):
+		table = '<div> <p style="text-align:center">' + measure + '</p>'
```
Comment (naslundx) [inline benchmarker.py:190]: Maybe this should be an `<h2>` or so, but design can wait until later.

**Comment by naslundx** [comment]
Great job.

File: benchmarker.py (hunk 2 above)
Comment (dekespo) [inline benchmarker.py:190]: Yes, we can design it later :)

**Pull request merged**